src/components/pagination/pagination.component.html

```diff
@@ -1,14 +1,18 @@
-<ul class="pagination" [ngClass]="classes" uxTabbableList direction="horizontal" [allowBoundaryKeys]="true">
+<ul #container class="pagination"
+  [ngClass]="classes"
+  direction="horizontal"
+  (blur)="isKeyboardEvent = false"
+  (keydown.ArrowLeft)="select(page - 1); isKeyboardEvent = true"
+  (keydown.ArrowRight)="select(page + 1); isKeyboardEvent = true">
```

Comment:
Could you add in the home/end functionality as well? I think that would be a convenient addition.